lib/plugins/aws/package/compile/events/apiGateway/lib/cors.js

```diff
@@ -30,9 +30,13 @@ module.exports = {
         'Access-Control-Allow-Origin': `'${origin}'`,
         'Access-Control-Allow-Headers': `'${config.headers.join(',')}'`,
         'Access-Control-Allow-Methods': `'${config.methods.join(',')}'`,
-        'Access-Control-Allow-Credentials': `'${config.allowCredentials}'`,
       };
 
+      // Only set Access-Control-Allow-Credentials when explicitly allowed (omit if false)
+      if(config.allowCredentials === true) {
```

Comment:
As in SLS `allowCredentials` is considered boolean, let's simply do `if(config.allowCredentials)` (checks as `if (value === true)` are not natural in JS.

Reply:
Ok, I could not find the place where you ensure a boolean. But this is fine for me.

My intention was to avoid having any truthy value considered as allowCredentials - true. For example:

```
// serverless.yml
allowCredentials: 'false' // would be interpreted as truthy now
```

Reply:
`'false'` is non empty string, so by JS boolean rules is a truthy value, and should be treated like that.

Still if someone will explicitly put `false` (boolean), then it's falsy, and will result with not listing that property. That's what's expected.

Reply:
Updated 👍